frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details-page.tsx

```diff
@@ -30,13 +30,17 @@ export const VMTemplateDetailsPage: React.FC<VMTemplateDetailsPageProps> = (prop
     disksPage,
   ];
 
-  const menuActions = undefined; // TODO(mlibra): menuActions
-
   const breadcrumbsForVMTemplatePage = (match: any) => () => [
     { name: VM_TEMPLATE_LABEL_PLURAL, path: `/k8s/ns/${match.params.ns || 'default'}/vmtemplates` },
     { name: `${match.params.name} Details`, path: `${match.url}` },
   ];
 
+  const menuActions = [
```

Comment:
could you please move these actions and VM template list actions to a separate file?

Reply:
sure :-) 

@mareklibra  @suomiy I think I will push the change before it merged ... :-)

Reply:
@yaacov its already in the merge pool, create a separate PR for it

Reply:
strange, it removed the lgtm but left the approved ... :unicorn: 

done, moved ... :-)

Reply:
I understand that the "approved" means the change itself makes sense for the project. The "lgtm" means the code is correct. So if new changes are detected, the "lgtm" is removed since the code changed and needs new re-evaluation. But it expects that you did not change it so much that it suddenly does not make sense for the project at all so leaves the "approved" in place :)